app/Keycloak/Repositories/EloquentKeycloakClientRepository.php

```diff
@@ -38,6 +39,8 @@ public function create(Client ...$clients): void
                             'realm' => $this->realms->getRealmByEnvironment($client->environment)->publicName,
                         ]
                     );
+
+                ClientCreated::dispatch($client->id);
```

Comment:
If you do this for every single Client in this loop, then I would move it to the model class.